subprocess_arm64.go:91:24: undefined: arch.ARMTrapFlag

```
$ bazel build --platforms=@io_bazel_rules_go//go/toolchain:linux_arm64 pkg/sentry/platform/ptrace/... --cpu=aarch64
INFO: Analyzed 2 targets (0 packages loaded, 0 targets configured).
INFO: Found 2 targets...
ERROR: /usr/local/google/home/avagin/git/gvisor/pkg/sentry/platform/ptrace/BUILD:5:1: GoCompilePkg pkg/sentry/platform/ptrace/linux_arm64_pure_stripped/ptrace%/gvisor.dev/gvisor/pkg/sentry/platform/ptrace.a failed (Exit 1) builder failed: error executing command bazel-out/host/bin/external/go_sdk/builder compilepkg -sdk external/go_sdk -installsuffix linux_arm64 -src pkg/sentry/platform/ptrace/filters.go -src pkg/sentry/platform/ptrace/ptrace.go -src ... (remaining 67 argument(s) skipped)

Use --sandbox_debug to see verbose messages from the sandbox
compilepkg: error running subcommand: exit status 2
/usr/local/google/home/avagin/.cache/bazel/_bazel_avagin/28f38c9ce4faaebb69a882e316b87bed/sandbox/linux-sandbox/7/execroot/__main__/pkg/sentry/platform/ptrace/subprocess_arm64.go:91:24: undefined: arch.ARMTrapFlag
INFO: Elapsed time: 0.759s, Critical Path: 0.47s
INFO: 3 processes: 3 linux-sandbox.
FAILED: Build did NOT complete successfully

```

--- a/pkg/sentry/arch/arch_aarch64.go
+++ b/pkg/sentry/arch/arch_aarch64.go
@@ -34,6 +34,9 @@ const (
 	SyscallWidth = 4
 )
 
+// ARMTrapFlag is the mask for the trap flag.
+const ARMTrapFlag = uint64(1) << 21
+
 // aarch64FPState is aarch64 floating point state.
 type aarch64FPState []byte
 

